.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "(Feature) Short description here"
+labels: enhancement
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Detailed Description
```

Comment:
```suggestion
## Description
```

Reply:
Actualy I think we whould keep the Detailed thing here. Its a project that might get more than the just us here so clarifytin "Detailed" might be a good idea

Reply:
We can keep it if you want as long as it's consistent among the templates; the other one has just **Description** at the moment.

Reply:
Good Point, I add that there as well. I used just stock temolate from here https://github.com/TalAter/open-source-templates